src/OracleService/OracleService.cs

```diff
@@ -345,9 +345,13 @@ private async Task<(OracleResponseCode, string)> ProcessUrlAsync(string url)
                 return (OracleResponseCode.Error, null);
             if (!protocols.TryGetValue(uri.Scheme, out IOracleProtocol protocol))
                 return (OracleResponseCode.ProtocolNotSupported, null);
+
+            using CancellationTokenSource ctsTimeout = new(Settings.Default.Https.Timeout);
```

Comment:
Https and Oracle have different timeout settings.

Reply:
Do we need to unify them?

Reply:
This issue should be fixed before merging.

Reply:
@shargon Can you fix it?

Reply:
I will do the changes